pkg/capabilities/mercury/mercury.go

```diff
@@ -55,8 +69,8 @@ type ReportInfo struct {
 
 // TODO: fix this by adding support for uint64 in value.go
```

Comment:
You can remove this by the way since FeedID was the field that was supposed to be uint